geopandas/tests/test_merge.py

```diff
@@ -105,6 +105,7 @@ def test_concat_axis1_multiple_geodataframes(self):
         reason="pandas <=1.0 hard codes concat([GeoSeries, GeoSeries]) -> "
         "DataFrame or Union[DataFrame, SparseDataFrame] in 0.25",
     )
+    @pytest.mark.filterwarnings("ignore:Accessing CRS")
```

Comment:
Do you know why this test case is raising the warning?

Reply:
Because after concatenating two GeoSeries without names we end up with a GeoDataFrame with no active geometry column. And `self._check_metadata` accesses `result.crs`, causing this warning to pop up. I believe this is intended behaviour here.